beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/hive/HiveClient.java

```diff
@@ -117,6 +117,9 @@ public boolean tableExists(String databaseName, String tableName) {
         return new HashMap<>();
       }
       return table.getParameters();
```

Comment:
could you put this at the end please? it took me a bit of time to realise about structure

Reply:
what was the resolution about this ? I am curious :)